pkg/controller/studyjob/katib_api_util.go

```diff
@@ -35,22 +36,33 @@ func initializeStudy(instance *katibv1alpha1.StudyJob) error {
 	if err != nil {
 		log.Printf("Connect katib manager error %v", err)
 		instance.Status.Condition = katibv1alpha1.ConditionFailed
-		return nil
+		return true, nil
 	}
 	defer conn.Close()
 	c := katibapi.NewManagerClient(conn)
+	_, err = c.GetStudyJob(context.Background(), &katibapi.GetStudyJobRequest{
+		StudyJobUid: string(instance.UID),
+	})
```

Comment:
in which case instance.UID can be duplicated?

Reply:
By CR operator implementation, the reconcile function will be called parallelly. So `initializeStudy` is often called two times. I don't know how to avoid it.